moveit_core/collision_detection/src/allvalid/collision_env_allvalid.cpp

```diff
@@ -32,118 +32,104 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Author: Ioan Sucan */
+/* Author: Ioan Sucan, Jia Pan, Jens Petit */
```

Comment:
Why are you adding Jia Pan as an author here?

Reply:
Because he is mentioned in `collision_robot_allvalid.cpp` see [here](https://github.com/ros-planning/moveit/blob/4ca54cd0cda1bc0ceef072d769357ce3802e62da/moveit_core/collision_detection/src/allvalid/collision_robot_allvalid.cpp#L35)